### DISCUSSION TITLE
gfx1032 "dimgray cavefish" navi 23 KDB files?

### DISCUSSION THREAD

**Discussion by ZeroTron** [ROCmSoftwarePlatform/MIOpen#1863] [Q&A]
Would it be possible to make some of these?  Also where does it know where to point to the file? Can't see it anywhere in rocm.

**Comment by atamazov** [comment]
@ZeroTron Do you need database that contains _pre-compiled_ kernels for Navi 23? Can you please let us know why you need it?

BTW maybe you'll find #1373 useful.
